distribution/changelog.txt

```diff
@@ -1,4 +1,4 @@
-0.1.2 (in development)
+﻿0.1.2 (in development)
```

Comment:
Please revert this change.

Reply:
(It looks like your editor inserted a byte order mark at the start of the file.)